ansible/roles/cartodb.app/tasks/main.yml

```diff
@@ -37,14 +37,43 @@
    args:
      chdir: "{{ cartodb_dir }}"
 
+ - name: Create a common data user
+   command: bundle exec rake cartodb:db:create_user QUOTA_IN_BYTES="{{ cartodb_common_data_user_data_quota }}" SUBDOMAIN="{{ cartodb_common_data_username }}" PASSWORD="{{ cartodb_common_data_user_password }}" EMAIL="{{ cartodb_common_data_user_email }}"
+   become_user: "{{ ansible_ssh_user }}"
+   args:
+     chdir: "{{ cartodb_dir }}"
+   when: cartodb_common_data_local
+
+ - name: Remove table limit for common data user
```

Comment:
Oh right, I had to add a rake task for that. Ok. I'm hesitant to use straight SQL queries for the user limits, in case the ruby tasks are doing more in response to limit changes.